apps/voting-disputable/app/src/components/VoteDetail/DisputablePeriod.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { GU, Timer, useTheme } from '@aragon/ui'
+import { format } from 'date-fns'
+
+//48 hour period
+const PERIOD = 1000 * 60 * 60 * 48
+const formatDate = date => `${format(date, 'yyyy-MM-dd, HH:mm')}`
+
+function DisputablePeriod({ startDate }) {
+  const theme = useTheme()
+  const now = new Date().getTime()
+  return (
+    <React.Fragment>
+      {now > startDate + PERIOD ? (
+        formatDate(startDate + PERIOD)
+      ) : (
+        <div
+          css={`
+            display: inline-flex;
+          `}
+        >
+          <Timer end={new Date(startDate + PERIOD)} />{' '}
```

Comment:
Do you think we should raise `startDate + PERIOD` into a named variable and use it in these three places?